fccodegen/_generator.py

```diff
@@ -341,9 +205,9 @@ def derivative_name(deriv):
 
     # Create output equation information dicts
     output_equations = []
-    output_symbols = [get_symbol_by_cmeta_id(graph, x) for x in outputs
+    output_symbols = [model.get_symbol_by_cmeta_id(x) for x in outputs
                       if x != 'state_variable']
```

Comment:
I take it `state_variable` is handled specially elsewhere?

Reply:
Yeah that's handled here: https://github.com/ModellingWebLab/fccodegen/blob/cellmlmanip-updates/fccodegen/_generator.py#L174-L180